### ISSUE TITLE
Staging Review finding: Focus remains on Continue button when there are errors

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124641]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** VA Form 21-0779
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Step 1 of 7 Nursing home official personal information, when clicking Continue when there are errors/empty required fields in the form, focus is remaining on the Continue button and not moving to the first field in error. Due to this, screen reader users may not be aware there there is an error nor understand how to move forward. 

Tested with Jaws in Chrome and Voiceover in Safari. 

Other occurences:
- occurs on most pages within the form

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="401" height="556" alt="error focus" src="https://github.com/user-attachments/assets/a829c9e5-d86b-42a5-88d2-fe2528e7a05b" />

</details>

## Recommended action

Ensure that focus is managed when clicking Continue, and that the first field in error is focused, so that screen reader and other AT users are aware that errors exist. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
